### PR TITLE
Fix warning prefixes

### PR DESCRIPTION
**Pull request by wbamberg**
This is part of https://github.com/mdn/content/issues/8180 (the last part, I hope).

It fixes up all warnings in Web/API so they are Markdown convertible. There weren't as many of these as the notes, so this is just one PR, but it's grouped by commits:

* https://github.com/mdn/content/commit/30a2b1b1fac4ccc3e6d211eb87b4430a88d2a243: convert prefix text from `Important` to `Warning`
* https://github.com/mdn/content/commit/63f4f247e83be860517e0460f329b9cdcbc99261: convert `<h4>`-style to `<strong>`-style
* https://github.com/mdn/content/commit/18ed8a27ec23f2b5680707f2eb62b56084d1e290: wrap warnings in `<p>`; move colon inside `<strong>` (2 changes in this one commit I'm afraid)
* https://github.com/mdn/content/commit/3d2e7045ef8fbf7c2a32cbb6f888106bd7d278b9: add `<strong>Warning</strong>` prefix where it was missing entirely
* https://github.com/mdn/content/commit/0d93699806e91e3fbc30daf50c02dc0032d3f78d: manual cleanup of ones I missed
* https://github.com/mdn/content/commit/6866a76fd97dcb2e5f98e2ff3d2aa149d8c0253d: finally I deleted a couple of pages that don't look useful. Only one was directly relevant to this change, but the other one came up in conversation and seemed worth removing too.
* https://github.com/mdn/content/pull/8244/commits/a9b0772c74ea012d320a5afcd928a825fc949fe9: remove links to the deleted pages

### CODE REVIEW

File: files/en-us/web/api/html_drag_and_drop_api/recommended_drag_types/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <p>The HTML Drag and Drop API supports dragging various types of data, including plain text, URLs, HTML code, files, etc. The document describes best practices for common draggable data types.</p>
 
 <div class="overheadIndicator nonStandard nonStandardHeader">
-<p><strong>Caution:</strong><br>
+<p><strong>Warning:</strong><br>
```
Comment (teoli2003) [inline files/en-us/web/api/html_drag_and_drop_api/recommended_drag_types/index.html:13]: Can you remove the `<br>`?

File: files/en-us/web/api/push_api/index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 <h2 id="Push_concepts_and_usage">Push concepts and usage</h2>
 
 <div class="warning">
-<p><strong>Warning: </strong>When implementing PushManager subscriptions, it is vitally important that you protect against CSRF/XSRF issues in your app. See the following articles for more information:</p>
+<p><strong>Warning:</strong>When implementing PushManager subscriptions, it is vitally important that you protect against CSRF/XSRF issues in your app. See the following articles for more information:</p>
```
Comment (teoli2003) [inline files/en-us/web/api/push_api/index.html:20]: Missing space after `<\strong>`

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Two small fixes are needed. Else, this is really cool.

File: files/en-us/web/api/html_drag_and_drop_api/recommended_drag_types/index.html (hunk 1 above)
Comment (wbamberg) [reply files/en-us/web/api/html_drag_and_drop_api/recommended_drag_types/index.html:13]:
```suggestion
<p><strong>Warning:</strong>
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**